content/development/developer-tools/index.en.yaml

```diff
@@ -0,0 +1,186 @@
+title: Developer Tools
+description: A collection of tools to enhance the Ethereum Classic application development experience
+disclaimer: true
+contribute: true
+intro: |
+  Good tools make a skilled craftsman better. This section contains a suite of excellent tools that can level up your development experience.
+
+  ## Clients
+
+  For more information on the various ETC clients, check out the [clients](/development/clients) page.
+tooling: | 
+  ## Development Tooling
+
+  A non-exhaustive selection of tools  that can aid development of Ethereum Classic applications.
+content:
+  - 
+    key: processors
+    type: buttons
+    title: Payment Processors
+    description: These APIs can be used to accept ETC payments in your website or application.
+    items:
```

Comment:
Content files can have have indentation, which is important to keep consistent in lists of items.